surround_cli/README.md

```diff
@@ -26,7 +26,7 @@ $ pip3 install surround_cli
 ### Generating projects
 For example you can use the sub-command ``init`` to generate a new project:
 ```
-$ surround init <path-to-dir> --project-name sample --description "Sample description" --require-web
+$ surround init <path-to-dir> --project-name sample --description "Sample description" --author-name <AUTHOR_NAME> --author-email <AUTHOR_EMAIL> --require-web <True/False>
```

Comment:
Do we need author name and email?

Reply:
Yep, for Poetry
It requires those information